types/node/index.d.ts

```diff
@@ -52,6 +52,16 @@
 
 /// <reference path="base.d.ts" />
 
+declare namespace NodeJS {
+    interface Dict<T> {
+        [key: string]: T;
+    }
+
+    interface ReadOnlyDict<T> {
+        readonly [key: string]: T;
+    }
+}
```

Comment:
This should probably be moved into a `types/node/globals.nodejs.dict‑type.d.ts` file, just like how `types/node/globals.global.d.ts` is handled: <https://github.com/DefinitelyTyped/DefinitelyTyped/blob/20efb58b919f1ff555b7137c57a337d20f90668f/types/node/globals.global.d.ts#L1>